docs/index.md

```diff
@@ -1,5 +1,89 @@
-Steps to set up an analysis pipeline to analyze base editor validation experiments using [CRISPResso2](https://github.com/pinellolab/CRISPResso2) 
-are enumerated below:
+Steps to set up an analysis pipeline to analyze base editor validation experiments to run [CRISPResso2](https://github.com/pinellolab/CRISPResso2): <br/>
+    1. Using the [BEV tool](https://gpplims.broadinstitute.org/screening/bev/create) on GPP LIMS (available to GPP only) ([Jump to steps](#running-bev-tool-on-gpp-lims)) <br/>
+    2. On your local machine ([Jump to steps](#running-crispresso2-on-local-machine))
+
+## Running BEV tool on GPP LIMS
+
+**Step 1: Sample name**
+
+Assign a descriptive name for the samples, without spaces and beginning with your initials (similar to PoolQ). 
+
+**Step 2: Construct file and Barcode file**
+
+* If sequencing available through WalkUp sequencing, you can use the Walk-Up Sample Finder to access sequencing files by inputting the sample ID and lane number.
+* If sequencing files are no longer available through WalkUp, the backed up files can also be accessed by providing a file path '/rnai/screening/pooled/sequenced/\[insert_filename_here.fastq.gz\]'
```

Comment:
Sequencing files can only be found in this path if they were run through PoolQ before, which might never be the case for BEV. A more generic path should be specified here.